### DISCUSSION TITLE
Separate Repos and Projects sections

### DISCUSSION THREAD

**Discussion by MehdiRtal** [arifszn/gitprofile#96] [Feature requests]
Would be better in my opinion if there's a separate section for showcasing your projects done separatly without having to publish them in GitHub as repos.

**Comment by arifszn** [comment]
Currently, there is no plan for this. As this project aims for GitHub users mainly, keeping it as it is.

Keeping this issue as discussion.

**Comment by arifszn** [comment]
Let's implement this.

**Reply by micheldore** [reply]
I've implemented this! Will make a pull request soon

**Reply by micheldore** [reply]
I've added the pull request! Let me know if there's any conflicts needing to be resolved.
